recipe/build.sh

```diff
@@ -1,14 +1,20 @@
 #!/bin/bash
 
 # Get an updated config.sub and config.guess
-cp -r ${BUILD_PREFIX}/share/libtool/build-aux/config.* .
+cp $BUILD_PREFIX/share/gnuconfig/config.* .
 
 # Avoid overlinking. zlib and liblzma are private to libxml2
 find "${PREFIX}/lib" -name "*.la" -delete -print
 sed -i.bak -e 's/-llzma //g' -e 's/-lz //g' $PREFIX/bin/xml2-config
 
+# It looks like an missing symbol of libxml on osx-arm64
+if [[ ${target_platform} =~ .*arm64.* ]]; then
```

Comment:
I think you could set this flag instead: --with-libxml-libs-prefix. Perhaps --with-libxml-include-prefix should be set too.
See: 
https://gitlab.gnome.org/GNOME/libxslt/-/blob/master/configure.ac#L437
https://gitlab.gnome.org/GNOME/libxslt/-/blob/master/configure.ac#L307

Reply:
```
The build  fails with flags:
            --with-libxml-include-prefix=$PREFIX/include/libxml2/ \
            --with-libxml-libs-prefix=$PREFIX/lib/
```